ci/travis/test_script.sh

```diff
@@ -30,7 +30,7 @@ if [[ $BUILD_DOCS == false ]]; then
   fi
 else
   cd doc
-  python make.py html -n 2
+  python make.py html
```

Comment:
No, but ironically, the parallel build doesn't change the run time :D 
We can easily activate the flags in sphinx-gallery to make it parallel-safe, but considering there is no gain at all, I haven't created the PR yet.
Actually making the code parallel requires more work, but should be doable.